report/src/presence_intro.tex

```diff
@@ -0,0 +1,23 @@
+\section{\label{sec:presence_intro}Room Occupancy Detection for Meeting Start and End Times}
+
+To address the problem defined in section \ref{TODO-SECTION 2.a.i AND 2.b.iii}, we propose an occupancy detection system (presence system) to identify when a meeting starts, defined as participants entering a room, and when it ends, defined as all participants leaving the meeting room.
+This section investigates various technologies for detecting room occupancy, evaluates their performance in the context of meeting start and end times, and determines the most suitable technology for our context of use.
```

Comment:
```suggestion
This section investigates various technologies for detecting room occupancy, evaluates their performance, and determines the most suitable technology for the context of the suggested system. 
```

Reply:
Why?

Reply:
How will you evaluate the performance of all the technologies in terms of 'context of meeting start and end times'?